### ISSUE TITLE
Increase 32 contour regions limit

### ISSUE THREAD

**Issue by dBitech** [hoche/splat#27]
It would be useful to have an output mode where the dBm or dBuV/m can have more than 32 steps. Given we output at least 24bit image files, we easily increase the number of contours. 

utilizing the same methodology as Mapbox terrain RGB tiles:
```
def signal_to_rgb(data, baseval, interval):
  """
  encode the signal level into uint8 RGB from any base and interval

  Parameters
  -----------
  data: ndarray  
    (RBG triplit) ndarray of data to encode

  baseval: float
    the base value of the RGB numbering system. will be treated as zero for this encoding

  interval: float
    the interval at which to encode

  Returns
  --------
  ndarray: (RBG triplit) data
      a uint8 (3 x rows x cols) ndarray with the data encoded
  """

  data = data.astype(np.float64)
  data -= baseval
  data /= interval

  rows, cols = data.shape

  datarange = data.max() - data.min()

  if _range_check(datarange):
    raise ValueError("Data of {} larger than 256 ** 3".format(datarange))

  rgb = np.zeros((3, rows, cols), dtype=np.uint8)

  rgb[2] = ((data / 256) - (data // 256)) * 256
  rgb[1] = (((data // 256) / 256) - ((data // 256) // 256)) * 256
  rgb[0] = ((((data // 256) // 256) / 256) - (((data // 256) // 256) // 256)) * 256

  return rgb
```

**Comment by der-stefan** [comment]
Absolutely! I fully agree with your idea. GeoTiff is already implemented in the refactor branch and would easily accept higher "color" depth or additional bands. The WriteImage functions have been also cleaned up and could be adapted to a universal output.

**Comment by der-stefan** [comment]
By the way: Use "-sc" for smooth rather than quantized colors. That should be default for path loss output in geotiffs anyway.

**Comment by dBitech** [comment]
I've done up a quick patch to allow the GeoTIFF to output band5 with the path loss,  does anyone think there ould there also be merit in adding the additional bands for Field strength?

**Comment by der-stefan** [comment]
My proposal: Let's split the outputs into several files:
  * *_raw.tif => contains pathloss and line-of-sight as two bands. This is a universal output independent of the TX power. You can take this file as input for the following outputs:
  * *_pl.tif => contains pathloss as RGBA with applied color palette
  * *_los.tif => contains line-of-sight as black pixels
  * *_dBm.tif => contains receive power as RGBA with applied color palette
  * *_dBuVm.tif => contains electric field strength as RGBA with applied color palette
